### ISSUE TITLE
Reopen Closed Item

### ISSUE THREAD

**Issue by itotallyrock** [mrodalgaard/atom-todo-show#124]
When pressing `Ctrl+Shift+T` on Windows it creates a pane where the Todo Show Pane would be with correct sizing but is completely blank and can't be closed.

Here's an image that shows the blank pane, not sure if this is on your end or Atom's but I've yet to have it occur with any other packages.

![](http://image.prntscr.com/image/fe3c23ccc8b7477ba05e32131d104ebd.png)

**Comment by itotallyrock** [comment]
Perhaps a better screenshot with more context.  The red box is the pane that was opened by pressing the hotkey to reopen a closed window.

![](http://image.prntscr.com/image/961d7a50c1414a65a551fea667b3f6f1.png)

When I press the bind for Todo Show it opens another Todo Show Pane.  As shown below.

![](http://image.prntscr.com/image/cbe44df03f504c8a9bbc59885f5caf2e.png)

**Comment by mrodalgaard** [comment]
Hi @itotallyrock, I saw something similar a while back after updating the package, but a reload or reopen of Atom fixed it. Also #109 was fixed a while back with something very similar. Are you using the newest version of Atom and the todo-show package?

The `flex-grow: .6` is perfectly fine. I'm guessing your problem comes from opening of new panes (more specifically the use of `atom.workspace.open` [HERE](https://github.com/mrodalgaard/atom-todo-show/blob/master/lib/show-todo.coffee#L115) and the destruction of the pane). If this is the case it is very strange that this issue is only happening for you.
